src/ec/suite_b/ops/ops.rs

```diff
@@ -338,7 +338,7 @@ pub struct PublicScalarOps {
 impl PublicScalarOps {
     pub fn scalar_parse(&self, input: &mut untrusted::Reader)
                         -> Result<Scalar, error::Unspecified> {
-        let encoded_value = try!(der::positive_integer(input));
+        let encoded_value = try!(der::positive_integer(input).map_err(|_| error::Unspecified));
```

Comment:
There were only a few places that needed to `map_err` so I chose to not implement `From<ASN1Error>`, but I can do that if you prefer.